docs/articles/immutability-in-js.md

```diff
@@ -0,0 +1,108 @@
+# Immutability in JavaScript
+
+According to (wikipédia)[https://en.wikipedia.org/wiki/Immutable_object] an *immutable object (unchangeable object) is an object whose state cannot be modified after it is created*. This rule is quite simple, if you want to modify some property of an object you have to do it on a copy. We will see a bit later what improvements and fancy features it unlocks for our developments.
+
+## EcmaScript
+
+EcmaScript provides utilities to keep our data immutable. APIs of arrays and objects have methods to create copies and even prevent instances to be updated. More recently EcmaScript brings us a brand new syntax that makes it easier to achieve it.
```

Comment:
`Recently EcmaScript brings us a brand new syntax that makes it easier to achieve.`

Reply:
`brings us` doesn't sound good, you could simply use `introduced`.
`brand new` sounds a little too much, `new` seems enough to me.
I think it's worth saying again what the syntax is for so:
`Recently ES introduced new syntaxes to create copies of objects and arrays.`.